the-collab-lab/tcl-69-smart-shopping-list
# [AM] Update home page

## Description

This update brings the home page in line with the team's Figma design (with some modifications):
![Screen Shot 2024-04-05 at 22 24 04](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/cc134aab-1c8e-492e-a5f7-5a98304864b8)

Specific additions and changes are laid out in AC below. These changes make the page more accessible, easier to quickly understand and use, and brings the feel of the page in line with current branding.

This update also adds a credit section at the bottom of the About page to credit Storyset for the illustration on the homepage.

## Related Issue

Closes #87 
Sub-issue of #14

## Acceptance Criteria
- [x] makes use of new brand fonts + colors
- [x] is easily viewable on smallest width screen size
- [x] buttons are larger/more prominent
- [x] rather than having text that directs user to buttons and their functions, make button text more directive (and ideally ~2 words) so they don't rely on surrounding text to identify their function
- [x] employs selected image from storyset (https://storyset.com/illustration/select/amico)

## Updates

### Before

![Screen Shot 2024-04-05 at 22 22 34](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/e1e196bb-abac-40d0-9f3b-865885ddcb80)
![Screen Shot 2024-04-05 at 22 22 24](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/aecaa11f-80d4-4950-bdc7-7aff1bdfa7e6)

### After

![Screen Shot 2024-04-05 at 22 45 32](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/608bee7d-f133-4b0c-b1d3-2a867c7447b3)
![Screen Shot 2024-04-05 at 22 39 55](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/867d6a72-499c-445c-ac7f-644302980034)
![Screen Shot 2024-04-05 at 22 39 44](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/115735046/7b988b7d-581e-4c42-818a-b8125d4f8109)

## Testing Steps / QA Criteria

Check that homepage appears as in designs above, both when signed in and signed out.
Check that bottom of About page now includes credit to Storyset for illustration.
